src/fdb5/toc/TocStore.cc

```diff
@@ -128,7 +128,7 @@ std::unique_ptr<FieldLocation> TocStore::archive(const Key &key, const void *dat
 
     ASSERT(len == length);
 
-    return std::unique_ptr<TocFieldLocation>(new TocFieldLocation(dataPath, position, length, Key(nullptr, true)));
+    return std::unique_ptr<TocFieldLocation>(new TocFieldLocation(dataPath, position, length, Key()));
```

Comment:
while at it, fix -> std::make_unique<TocFieldLocation>(dataPath, position, length, Key())

Reply:
You can do. It doesn't really matter (the only case where std::make_unique makes a difference is if you have a constructor to an object which owns multiple dynamically allocated objects, where the constructor of one of those objects throws after the other has been constructed, and you want to ensure there is no memory leak.